1.2.0/index.html

```diff
@@ -292,191 +280,263 @@ <h2>Terminology</h2>
+<p class="note">
+CDXJ's name name and semantics partly derive from an earlier index format
+developed as part of the Internet Archive's <a>Wayback Machine</a>, where CDX
+may have been an acronym for Crawl (or Capture) inDeX. WACZ's implementation of
+CDXJ also draws on a proposed, but never implemented, WARC index format for the
+OpenWayback project [[OPENWAYBACK-CDXJ]]. 
+</p>
 
+A CDXJ file is a sorted, line oriented plain-text file (optionally GZIP
+compressed) where each line represents information about a single capture in a
+web archive collection.
 
-- `mainPageURL`: An optional URL of the main or starting page in the collection,
-if any, to be used for initial replay.
+Each line MUST have three components that are separated by single spaces (0x20):
 
-- `mainPageDate`: An optional ISO-formatted date of the main or starting page in the collection, if any, to be used for initial replay. Specified only if `mainPageURL` is specified.
+1. a Searchable URL
+2. an Integer Timestamp
+3. a JSON Block 
 
-- "software": A description of what software was used to create the WACZ file
+### Searchable URL
 
-### datapackage-digest.json 
+The Searchable URL is a normalized form of the archived URL that allows a
+CDXJ file to be sorted and efficiently scanned using a binary search algorithm.
+The Searchable URL is sometimes referred to as Sort-friendly URI Reordering
+Transform (SURT).
 
-With WACZ 1.1, there is now also support for a special *datapackage-digest.json*, which makes it possible to verify the *datapackage.json* manifest
-with a hash, and an optional signature, and thus for the entire contents of the WACZ.
+The steps for creating a Searchable URL from a URL are:
 
-The `hash` and `path` keys are required, while `signature` and `publicKey` are optional.
+1. lowercasing the URL
+2. removing the protocol portion (HTTP or HTTPS)
+3. replacing the host name portion of the URL with a reversed, comma separated
+equivalent: `www.example.org becomes `org,example,www`
+4. adding a `)` separator
+5. adding the remaining portion of the URL (path and query)
 
-**TODO: align with current archiveweb.page implementation?**
+For exapmle the URL ```https://www.example.org/index.html``` would be
+represented as this Searchable URL ```org,example,www)/index.html```
 
-Ex:
-  ```json
-  {
-    "hash": "sha256:...",
-    "path": "datapackage.json",
-    "signedData": {
-    "signature": "...",
-    "publicKey": "..."
-  }
-  ```
+### Integer Timestamp
 
-## Record Digests
+The Integer Timestamp is an integer representation of the date and time (UTC) when the
+web archive snapshot was created. It is composed of:
 
-With WACZ 1.1, index entries for each individual WARC record in the index (CDX) includes a digest of the WARC record.
+- 4 digit year (e.g. 2022)
+- 2 digit month (e.g. 02)
+- 2 digit day (e.g. 05)
+- 2 digit hour in 24 hour format (e.g. 23)
+- 2 digit minute (e.g. 13)
+- 2 digit second (e.g. 59)
+- 3 digit milliseconds (e.g. 032)
```

Comment:
Could say that the milliseconds are optional, could support either a 14-digit or 20-digit timestamp. WARC/1.0 only uses 14-digit, while WARC/1.1 supports upto 20.

Reply:
Ok I've updated the milliseconds to use MAY. We might want to add some more bits about how this is derived from WARC?